packages/engine/README.md

```diff
@@ -203,14 +203,20 @@ In order to see more logging information while the simulation is running, you ca
 $ export RUST_LOG=debug
 ```
 
+If you only want to see the output from your behaviors, you may set
```

Comment:
It's `info` by default right? I'm not sure about the value in this statement. It also doesn't only output the logs from the behaviors, there are logs about the experiment starting, simulations finishing, persistence, etc.

I would hope that the default, or someone setting it to `info` would be relatively self-explanatory in that it's showing less logs (especially as we link to the Rust docs)

Reply:
`info` is the default, but we are setting `debug` for `#[cfg(debug_assertions)]` if `RUST_LOG` is not set. This instruction **only** prints the info-output generated from `cli::experiment`.

Reply:
Gotcha, I've just tried it out locally and properly understand. To be honest I'm still not sure I see the point in us putting this in the README. I also don't think the setting is particularly robust because it only works right now because we don't have another `log::info!` in `cli::experiment` it's not like we've manually made a toggle to only expose the logs from the behavior code

Reply:
I agree. With [Improve logging traceability](https://app.asana.com/0/1199550852792314/1201545236176390/f
) we will come up with a better solution.